pkg/ebpf/tracer-ebpf.go

```diff
@@ -69,7 +69,7 @@ func (fi bindataFileInfo) Sys() interface{} {
11\xe1\x61\x39\x0f\xe7\x47\xaa\xab\xd0\x44\x55\xd0\xab\x0a\x7a\x55\xa1\xa9\xaa\x84\x7e\xc8\x58\x85\x84\x2a\x91\x50\x85\x92\xc2\x28\x21\x8c\x8c\x21\xc4\x43\x28\x31\x04\x46\x1e\x96\x8b\x30\x98\xca\x50\x21\x54\xa9\x84\x0a\x95\x50\xa1\x52\xa8\x80\x84\x70\x25\x9a\xa8\x12\x92\x2a\x21\x09\x0c\x3c\x2c\x17\x61\x30\x95\xb1\x02\x19\x2b\x90\xb1\x02\x19\x2b\x44\xc6\x8a\x4a\xd7\xb6\x0b\xa3\xad\xc2\x50\x30\x8c\xb6\x0a\x43\xc1\xb0\x50\x10\x19\xc2\xc8\x10\x46\x86\xb0\xc8\x10\x46\x49\xa1\x30\xda\x04\x19\x6b\x42\xa8\x72\x08\xe9\x21\xd4\x2c\x84\x9a\x21\x9d\x87\x61\x84\xe5\x22\x0c\xe2\xc7\xfc\x48\x4d\x05\x0a\xac\xe0\x0c\x3c\xac\x8e\x54\x57\x40\xd5\x0a\xa1\x6a\x05\x24\x21\x43\x08\x8c\x21\x30\x86\xc0\xc0\xc3\x72\x11\x06\xf1\x23\x14\xa9\x40\x01\x35\xe5\xbc\x00\x1e\x56\x21\xac\x44\x18\x46\x38\x3f\x12\x42\xbe\x10\xf2\x85\x90\x2f\x84\x7c\x3c\x2c\x17\x61\x10\x3f\xe6\x47\x6a\x82\x10\x10\x84\x80\x20\x04\x04\x21\x20\x08\x06\xfe\x63\x7e\x24\x8c\x30\x04\xc6\x10\x18\x43\x60\x0c\x81\x91\x87\xe5\x22\x0c\x26\x7f\x4c\xc4\xf3\xcf\xf8\xce\x9a\x3e\x85\xd8\x50\xe5\x10\xdc\x8c\xc2\xc2\x06\xfe\x11\xd8\x2d\xb5\xbf\xa5\x0f\xfd\x24\xe9\x74\x23\x3d\x3f\xc5\xf1\x89\x3a\x6c\xa6\xc1\x5c\xbd\x22\xc1\xc7\x79\x28\xe0\xd5\x4b\x70\x4f\xbe\xe0\x17\xf4\x43\x1e\xfa\x8b\xfd\x1d\xfd\xce\x80\xca\x7f\x9d\xb9\xcb\xaf\x7d\xdc\x59\x8f\x4b\xe0\x8a\xa6\xf2\xff\xd3\x43\x7e\x1f\xff\x4c\x37\x63\xfd\xbf\xe5\x21\xbf\xd3\xa5\xfe\xf9\x2e\xf2\x6f\xf3\x90\x5f\x0a\xd7\x05\xfd\x4e\x82\xca\x4f\x78\xc8\xef\x72\xa9\xff\x34\x17\xf9\xb7\xfa\xdc\xe5\x6f\x82\xbf\x8f\x7e\x87\x41\xe5\xdf\xe2\x73\x97\x3f\x04\xf9\x51\x9b\xfc\x19\x2e\xf2\x1f\xf0\x90\xdf\x7f\x31\x0f\xe9\x77\x1e\x54\xfe\xf7\x3c\xe4\xd7\x3e\x21\xcb\x3f\xc5\x45\xfe\x1f\x3d\xe4\xd7\x5f\xc5\x43\xfa\x1d\x09\x95\xff\x3b\x0f\xf9\x7d\x90\xdf\x67\x93\x3f\xd3\x45\xfe\x27\x3c\xe4\x0b\x87\x01\xfd\x4e\x85\xca\xbf\xd1\x43\x7e\xfd\x93\x3c\xb4\xd7\x7f\x96\x8b\xfc\x90\x87\xfc\xd2\x98\x93\xdf\x4b\xfe\x65\x5e\xfd\xff\xa4\x53\x4e\x52\xfe\xe9\x2e\xf2\xe7\x7a\xc8\xdf\x0a\xf9\xf4\x3b\x1b\x2a\xbf\xc8\x43\xfe\x30\xe4\x77\xd9\xe4\x17\xb9\xc8\xbf\xc2\xef\x2e\xbf\xf3\x76\xf0\x07\xd2\xcb\xaf\xf1\xbb\xcb\x8f\xfe\x0c\xa1\x4d\xfe\x1c\x17\xf9\x97\x79\xcc\xff\xc2\x7b\x78\x48\xbf\x03\xa2\xf2\x2f\xf6\x98\xff\xc3\x90\x1f\xb4\xc9\x9f\xeb\x22\x7f\xaf\xc7\xfa\x5f\x7f\xaf\x9a\xfc\x7e\x8f\xf5\xbf\xf0\x29\x59\xfe\x99\x2e\xf2\x1b\x20\x9f\xda\xc0\xbe\xfb\x78\x48\xcf\x3d\x50\xfb\xf5\x31\x0f\xfe\xc1\x6f\xab\xf1\x5f\xe1\xc1\x3f\xdc\xa7\xc6\x9f\x97\xe7\xce\x3f\x7a\xbf\x1a\xff\x63\x39\xee\xfc\x85\xdf\x55\xe3\xff\x95\x87\xfe\xa5\xbb\xd5\xf8\x3b\x3d\xf8\x6b\xbf\xa7\xc6\x7f\x91\x07\xff\xe2\xef\xab\xf1\xaf\xf5\xe0\x8f\xee\x51\xe3\xdf\xe2\xc1\xbf\xa9\xdf\x3d\x3f\x8d\xff\xcd\xe7\xce\xff\x05\x0f\x7e\xba\xff\x5a\xe6\x77\xff\x3e\xb5\x0b\xfc\x14\x71\xa1\x79\xf3\x3d\xbe\x6f\xbd\x0b\xfc\x25\x19\xf8\x73\xfd\xd6\xb7\xd5\xf6\x67\x08\xfc\x7d\x19\xe6\xaf\xe1\xb1\x7e\xd5\x62\xfe\x0a\xbf\x5e\x72\xfe\x96\xb9\xd9\x2f\xbf\x2c\x3b\xf9\x74\x3e\x08\xf9\x36\x3f\xe5\x1d\x36\x7e\xb1\xac\xfe\x5f\x00\x00\x00\xff\xff\xd5\x6b\x90\x17\x58\xe3\x00\x00")
```

Comment:
why did this change?

Reply:
`clang-8` generates a different byte code than the older clang version we were using